ansible/roles/ocp4-workload-integreatly/sample_vars/ocp4_workload_integreatly_vars.yml

```diff
@@ -2,4 +2,12 @@
 # override ocp workloads to include prerequisite roles
```

Comment:
Does this directory have to be named `sample_vars`? Maybe just `vars` would be better?

Reply:
@dimitraz As far as I'm aware, `vars/` is a "standard" or "expected" directory name in a role for Ansible. I'd be tempted to keep it this way unless there's a good reason not to, to make it clear this is just a local dev example

Reply:
Ok that makes sense. 
How are these workloads run sequentially then in an actual Integreatly PDS workshop? Is there some tower job that we have to create? If so where does that have to be created?
Sorry for all the questions, just not too familiar with this workflow

Reply:
When we set up a config we provide an ordered list of workloads to be applied after the OpenShift installation has finished. So in your example it would call the authentication workload first, followed by minio, followed by whatever else you need in whatever order.

That is a setting in the configuration for RHPDS.